From 6ca7a2a19ea2431be43a98c33fd2dd289e2b2bc0 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Thu, 8 Dec 2022 11:07:25 +0100
Subject: [PATCH 0383/1141] dt-bindings: pci: stm32: add aer optional
 interrupts

The synopsys controller integrated in STM32MP25 can manage the AER MSI
through platform interrupt line connected to the GIC. Hence describe this interrupt

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I7bc8edcf51ee08b1963ed9fe6679d9d9d9a0e0e5
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/309512
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 .../devicetree/bindings/pci/st,stm32-pcie-host.yaml    | 10 ++++++++++
 1 file changed, 10 insertions(+)

--- a/Documentation/devicetree/bindings/pci/st,stm32-pcie-host.yaml
+++ b/Documentation/devicetree/bindings/pci/st,stm32-pcie-host.yaml
@@ -46,6 +46,14 @@ properties:
   clock-names:
     const: core
 
+  interrupts:
+    description: AER interrupts
+    maxItems: 1
+
+  interrupt-names:
+    items:
+      - const: aer_msi
+
   phys:
     maxItems: 1
 
@@ -111,6 +119,8 @@ examples:
                              <0 0 0 2 &intc 0 0 GIC_SPI 265 IRQ_TYPE_LEVEL_HIGH>,
                              <0 0 0 3 &intc 0 0 GIC_SPI 266 IRQ_TYPE_LEVEL_HIGH>,
                              <0 0 0 4 &intc 0 0 GIC_SPI 267 IRQ_TYPE_LEVEL_HIGH>;
+             interrupts = <GIC_SPI 232 IRQ_TYPE_LEVEL_HIGH>;
+             interrupt-names = "aer_msi";
              #address-cells = <3>;
              #size-cells = <2>;
              ranges = <0x01000000 0 0x10010000 0x10010000 0 0x10000>,
